webui/react/src/pages/DesignKit.module.scss

```
@@ -1,10 +1,13 @@
+/* stylelint-disable no-descending-specificity */
+
 .base {
   display: flex;
   gap: 8px;
   height: 100%;
   overflow: auto;
 
-  & nav {
+  & :global(nav:not(.ant-breadcrumb)) {
```

```suggestion
    & nav:not(:global(.ant-breadcrumb)) {
```

i think its more accurate